piker/brokers/kraken/feed.py

```diff
@@ -0,0 +1,464 @@
+# piker: trading gear for hackers
```

Comment:
Everything here was again moved over verbative from the old *god* module..

Reply:
lol what a module name